src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  list-style: none;
+  margin: 0;
+    
+}
+
+.nav__item {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0;
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  cursor: pointer;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 14px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
would be good to move this color to variable

Reply:
I must to make a variable only here or also in other tags where this color is present?